common/authorization/oauthAutorizer_test.go

```diff
@@ -108,23 +108,23 @@ func (s *oauthSuite) TearDownTest() {
 
 func (s *oauthSuite) TestCorrectPayload() {
 	authorizer := NewOAuthAuthorizer(s.cfg, s.logger)
-	result, err := authorizer.Authorize(s.ctx, &s.att)
+	result, err := authorizer.Authorize(s.ctx, &s.att, nil)
```

Comment:
You might want to add a test which passes the domain cache

Reply:
yeah I will ping you again once ready.